src/components/AssetSearch/AssetList.tsx

```diff
@@ -23,17 +24,69 @@ type ItemData<T> = {
   handleClick: T
 }
 
+const sortAssetsByAccountAndMarketCap = (
+  assets: Asset[],
+  enrichedData: Record<string, { fiatAmount: string; cryptoAmount: string; marketCap: string }>,
+): Asset[] => {
+  return sortBy(assets, [
+    asset => {
+      return Number(enrichedData[asset.assetId].fiatAmount)
+    },
+    asset => {
+      return Number(enrichedData[asset.assetId].marketCap)
+    },
+  ]).reverse()
+}
```

Comment:
Nitpick: use return shorthand
Not a nitpick: Don't use `Number` primitive wrapper to cast to an int, bignumber.js has methods for that
```suggestion
const sortAssetsByAccountAndMarketCap = (
  assets: Asset[],
  enrichedData: Record<string, { fiatAmount: string; cryptoAmount: string; marketCap: string }>,
): Asset[] =>
  sortBy(assets, [
    asset => bnOrZero(enrichedData[asset.assetId].fiatAmount).toNumber(),
    asset => bnOrZero(enrichedData[asset.assetId].marketCap).toNumber(),
  ]).reverse()
```

Still not the happiest with using actual JS numbers since we're dealing with BNs, but from testing it out this works so I'm happy with it for now.
We can take this over and iterate on it later on with an alternative implementation e.g using buckets like in https://github.com/shapeshift/web/pull/1304